packages/styles/src/sass/components/footer/_external.scss

```diff
@@ -0,0 +1,176 @@
+@use './internal' as *;
+@use '../../tokens/breakpoints' as *;
+
+.cbp-external-footer {
+  & > section:first-child {
+    display: flex;
+    flex-direction: column;
+
+    & > .cbp-footer__heading {
+      background-color: var(--cbp-color-base-secondary-dark);
+      padding: var(--cbp-space-3x) 0 var(--cbp-space-3x) var(--cbp-space-5x);
+    }
+  }
+
+  & > .cbp-footer__info {
+    row-gap: var(--cbp-space-8x);
+  }
+}
+
+.cbp-external-footer__social {
+  @include footer-nav-bar;
+  display: flex;
+  padding-left: var(--cbp-space-5x);
+
+  a[href] {
+    font-size: var(--cbp-font-size-heading-md);
+    padding: 0.743rem; // Due to font-size, using this rem value is closest to height of 44px and width ~44px in mockups
+  }
+}
+
+.cbp-external-footer__seal {
+  align-items: center;
+  display: flex;
+  margin-bottom: var(--cbp-space-4x);
+
+  & > span {
+    color: var(--cbp-color-text-lightest);
+    font-size: var(--cbp-font-size-heading-xl);
+    font-weight: var(--cbp-font-weight-bold);
+    margin-left: var(--cbp-space-2x);
+  }
+
+  // Link Component Dark Stylings
+  & + p > a[href] {
+    @include footer-links;
+  }
+}
+
+.cbp-external-footer__links {
+  font-size: var(--cbp-font-size-heading-xs);
+  font-weight: var(--cbp-font-weight-medium);
+  list-style: none;
+  margin: 0;
+  padding: 0;
+
```

Comment:
I think if you set columns to 2 or 3 and column-width: 8rem that you get a good balance without media queries. Then you can delete these properties from the larger media queries and let them inherit from here.

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Columns/Basic_Concepts_of_Multicol#specifying_both_number_and_width_of_columns